src/app/public/modules/datepicker/fuzzy-date.service.spec.ts

```diff
@@ -18,16 +28,136 @@ const moment = require('moment');
 
 describe('SkyFuzzyDateservice', () => {
   let service: SkyFuzzyDateService;
+  let currentLocale: string;
   const defaultDateFormat = 'mm/dd/yyyy';
+  const appProvider = new SkyAppLocaleProvider();
 
-  beforeEach(() => {
+  beforeEach(fakeAsync(() => {
     TestBed.configureTestingModule({
       providers: [
+        SkyAppLocaleProvider,
+        SkyAppWindowRef,
         SkyFuzzyDateService
       ]
     });
 
     service = TestBed.get(SkyFuzzyDateService);
+    tick();
+
+    appProvider.getLocaleInfo()
+    .take(1)
+    .subscribe((localeInfo) => {
+      currentLocale = localeInfo.locale;
+    });
+  }));
+
+  afterEach(() => {
```

Comment:
Does this not happen automatically?

Reply:
Oops. That was a sanity check that I should have removed before committing. Thanks for catching that! Removed.